### ISSUE TITLE
Sorting the list of awesome-python-modules

### ISSUE THREAD

**Issue by prashant-shahi** [gauravssnl/awesome-python-modules#1]
Sorting the list of awesome-python-modules with their different categories would make it an ease to browse through.

**Comment by gauravssnl** [comment]
Yes, categorization will make browsing easier, but a lot of time is required to do that . Another reason is that categorization is not an easy task . For example, some of the modules may be related to several categories.
I made this repository from my Facebook notes where I just stored GitHub repository URL & details , but no categories. The number of repository links are large.
